VideoAPI/Video.API/Controllers/ParticipantsController.cs

```diff
@@ -266,5 +266,20 @@ public async Task<IActionResult> SaveHeartbeatDataForParticipantAsync(Guid confe
 
             return NoContent();
         }
+
+        /// <summary>
+        /// Get a list of distinct first name of judges
+        /// </summary>
+        /// <returns></returns>
+        [HttpGet("participants/judge/firstname")]
```

Comment:
judge should be plural since it's a collection of resources

Reply:
Fixed